great_expectations/experimental/datasources/interfaces.py

```diff
@@ -222,15 +223,11 @@ def _validate_batch_request(self, batch_request: BatchRequest) -> None:
         )
 
     # Sorter methods
-    @pydantic.validator("order_by", pre=True, each_item=True)
-    def _parse_order_by_sorter(
-        cls, v: Union[str, BatchSorter]
-    ) -> Union[BatchSorter, dict]:
-        if isinstance(v, str):
-            if not v:
-                raise ValueError("empty string")
-            return _batch_sorter_from_str(v)
-        return v
+    @pydantic.validator("order_by", pre=True)
+    def _parse_order_by_sorters(
```

Comment:
@NathanFarmer FYI: I am going to attempt to move the job of sorting into `DataConnector` (not sure if this code will change.  Thanks.